tests/introspection/ApiTypoTest.cs

```diff
@@ -293,6 +303,7 @@ protected ApiTypoTest ()
 			"Mult",
 			"Multipath",
 			"Multipeer",
+			"Musthold",
```

Comment:
Can't this go in the `!XAMCORE_4_0` section below?

Reply:
Ahh yah, I added a couple of these before I did the !XAMCORE_4_0 section.  Will move